Automattic/wp-calypso
# Hosting: Update Dialog for phpMyAdmin Password Reset

Fixes #95889 

## Proposed Changes

* Updates the "Reset password" confirmation dialog for the phpMyAdmin card. 

## Why are these changes being made?

The current dialog design is inconsistent with others. Most importantly, the buttons are the wrong way round, which can cause some confusion, but some of the spacing is also off. In addition, we have a `ConfirmDialog` component specifically for this purpose, so it makes sense to use it. 

## Testing Instructions

* Go to `/hosting-config/site.com` on an Atomic site 
* Navigate to Server Settings
* Click "Try resetting the password" under the phpMyAdmin card
* Confirm the new dialog works

| Before | After |
|--------|--------|
| <img width="616" alt="Screenshot 2024-10-31 at 23 06 44" src="https://github.com/user-attachments/assets/7ac06cbf-bcc7-450b-865f-b5295d1c4717"> | <img width="698" alt="Screenshot 2024-10-31 at 23 01 43" src="https://github.com/user-attachments/assets/100e37ec-4829-40ff-ac52-47f06595c0f0"> | 

cc @nightnei, @wongasy, @fushar
